src/js/lib/client/src/preload.js

```diff
@@ -1,4 +1,14 @@
-let win = require('electron').remote.getCurrentWindow()
+const remote = require('electron').remote
+
+let win;
+if (remote !== undefined){
+    win = remote.getCurrentWindow()
+}
+
+if (win === undefined) {
+    console.warn("Could not get current window object for animation during preload.")
+}
+
```

Comment:
The error should be more specific - instead fo "animation during preload" it should tell "window startup animation"